.gitignore

```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+*lock.json
```

Comment:
Why did you remove this file and decided to ignore?
package-lock.json is an important part of any project/package